packages/mixer-connection/src/lib/facade/fx-bus.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { firstValueFrom } from 'rxjs';
+import { SoundcraftUI } from '../soundcraft-ui';
+
+describe('FX Bus', () => {
+  let conn: SoundcraftUI;
+
+  beforeEach(() => {
+    conn = new SoundcraftUI('0.0.0.0');
+  });
+
+  it('fxType$', async () => {
```

Comment:
😮 You can set every effect to be anyone?! OMG

I need to test if the effects work.

ℹ️ _Print from the demo web page... I'm not in my studio right now_
![Captura de Tela 2024-10-18 às 10 22 31](https://github.com/user-attachments/assets/57e84df5-316f-46dd-be4a-4cae505e7fb0)

EDIT: the values of the test changed since this comment was created

Reply:
We can set the value and the Web App shows the right params, but will the FX processor actually work? Can you try this out with your hardware?